### PR TITLE
[PPP-11141] Initial Documentation

### PR DESCRIPTION
**Pull request by jlam-mpc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,10 +40,176 @@ Setup prisma (and restart development server)
...
 For example:
 
-    feature/my-feature-branch-name
\ No newline at end of file
+    feature/feature-branch-name
+
+## Pull Request Naming Conventions
+
+Please follow the PR standard below:
+- enclose last 5 scrum task number with with `[PPP-#####]`
```
Comment (elizabeth-jimenez) [inline README.md:57]: double 'with'

File: README.md [hunk 2]
```diff
@@ -30,10 +40,176 @@ Setup prisma (and restart development server)
...
+- include lowercase short description
+
+For example:
+
+    [PPP-11001] example short description
```
Comment (elizabeth-jimenez) [inline README.md:62]: 🙌

File: README.md [hunk 3]
```diff
@@ -30,10 +40,176 @@ Setup prisma (and restart development server)
...
+## Code
+
+Document complex components with JSDocs and in-line documentation to assist other developers in understanding logic and purpose.
+
+Example:
+```tsx
+/**
+ * This is an example comment block that turns into JSDocs.
+ * It is viewable in the code and when the developer hovers
+ * over any usage of 'parseData' throughout the code.
```
Comment (elizabeth-jimenez) [inline README.md:117]: should this be `over any usage of 'percentify'`?

File: README.md [hunk 4]
```diff
@@ -30,10 +40,176 @@ Setup prisma (and restart development server)
...
+- Field default constraints
+- Enumerations
+- Sync logic
+- Explain how data is maintained
+
+### Forecast
+
+| Name | Type | Constraints | Description |
+|---|---|---|---|
+| id |  | PRIMARY_KEY |  |
```
Comment (elizabeth-jimenez) [inline README.md:141]: 😭 ❤️

**Comment by elizabeth-jimenez** [comment]
Couple comments but Approved otherwise!

File: README.md (hunk 3 above)
Comment (jlam-mpc) [reply README.md]: Fixed in recent commit!

File: README.md (hunk 1 above)
Comment (jlam-mpc) [reply README.md]: Fixed in recent commit!

**Pull request merged**